11ty/definitions/antisemitism.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Antisemitism
+slug: antisemitism
+defined: true
+speech: noun
+excerpt: aggression towards, discrimination, or prejudice against Jewish people.
+reading:
+   - text: "Antisemitism Education, Training &amp; Thought Leadership, from Jews for Racial &amp; Economic Justice"
+     href: https://www.jfrej.org/assets/uploads/JFREJ-Understanding-Antisemitism-November-2017-v1-3-2.pdf
+   - text: "Working definition of antisemitism from The International Holocaust Remembrance Alliance (IHRA)"
+      href: https://www.holocaustremembrance.com/working-definition-antisemitism
+skip_in_table_of_content: true
+---
+
+aggression towards, discrimination, or prejudice against Jewish people.
+
+Antisemitism can arise in many different ways, including:
+
+-  racial, which implies that Jews constitute a distinct race or ethnic group with inherent traits or negative characteristics;
+-  economic, which infers certain activities become harmful when performed by Jews, often specifically links Jews and money;
+-  religious, which includes discrimination against Jewish religious beliefs and religious teachings by other groups implying the inferiority of Jews;
+-  and political, which implies Jews seek national and/or world power.
+
+Antisemitic ideology includes dehuminization and degredation of Jews, as well as myths which adapt over time.
+
+## How it works
+
+Many groups are oppressed due to a fixed hierarchy, such as anti-Black racism in the United States. Antisemitism is part of a repeating cycle which goes in waves of severity.
+
+Antisemitism is most visible when people are forced to compete for scarce resources.
+
+## Origin
+
+Antisemitism began in European Christianity as ideological oppression, and functions to protect the ruling class and divert blame for hardship. While it began as a religious intolerance, antisemitism has always in-part been xenophobic.
+
+#### Note
+
+The International Holocaust Remembrance Alliance (IHRA) addresses the [spelling of 'antisemitism'](https://www.holocaustremembrance.com/antisemitism/spelling-antisemitism), instead of 'anti-Semitism':
```

Comment:
```suggestion
The International Holocaust Remembrance Alliance (IHRA) addresses the [spelling of 'antisemitism'](https://www.holocaustremembrance.com/antisemitism/spelling-antisemitism), instead of 'anti-Semitism':
```
Same removal here for IHRA:
https://www.noihra.ca/